pkg/assume/assume.go

```diff
@@ -578,8 +578,8 @@ func AssumeCommand(c *cli.Context) error {
 				clio.Warn("No credential suffix found. This can cause issues with using exported credentials if conflicting profiles exist. Run `granted settings export-suffix set` to set one.")
 			}
 
-			clio.Successf("Exported credentials to ~/.aws/credentials file as %s successfully", profileName)
-
+			credentialsFilePath := cfaws.GetAWSCredentialsPath()
```

Comment:
This made for the smallest PR, but if preferred I could offer a version in which `cfaws.ExportCredsToProfile()` returns the path so this is less implementation-aware.